.changesets/config_simon_router_version_in_cache_key.md

```diff
@@ -0,0 +1,7 @@
+### Distributed query plan cache keys include the Router version number ([PR #6406](https://github.com/apollographql/router/pull/6406))
+
+If you have enabled [Distributed query plan caching](https://www.apollographql.com/docs/router/configuration/distributed-caching/#distributed-query-plan-caching), starting with this release and going forward you should anticipate additional cache regeneration cost when updating between any Router versions.
```

Comment:
Should this discuss the possibility a staged rollout to start warming up new cache entries while the previous version still serves most traffic? I didn’t want to sound like "draw the rest of the owl"

Reply:
I think "drawing the rest of the owl" can be added to distributed query plan caching docs themselves. It will be a nice addition, and people can search the docs with a bit more ease than a changelog.